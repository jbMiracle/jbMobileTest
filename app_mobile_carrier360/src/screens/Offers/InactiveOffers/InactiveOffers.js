import React, { Component } from 'react';
import { Text, View } from 'react-native';

class InactiveOffers extends Component {
  render() {
    return (
      <View testID='offers' style={ { flex: 1, justifyContent: 'center', alignItems: 'center' } }>
        <Text>Inactive Offers</Text>
      </View>
    );
  }
}

export default InactiveOffers;
