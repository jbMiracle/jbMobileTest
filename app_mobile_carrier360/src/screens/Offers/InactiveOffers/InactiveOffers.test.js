import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import InactiveOffers from '../InactiveOffers/InactiveOffers';

it('can render an empty component', () => {
  const renderer = new ShallowRenderer();

  renderer.render(<InactiveOffers />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
