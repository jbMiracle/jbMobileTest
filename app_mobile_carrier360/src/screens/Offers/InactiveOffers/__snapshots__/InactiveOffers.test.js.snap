// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an empty component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="offers"
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Inactive Offers
  </Text>
</View>
`;
